pkgs/development/python-modules/nominal/default.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  requests,
+  nominal-api,
+  nominal-api-protos,
+  python-dateutil,
+  conjure-python-client,
+  pandas,
+  typing-extensions,
+  click,
+  pyyaml,
+  tabulate,
+  ffmpeg-python,
+  coverage,
+  pytest-cov-stub,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "nominal";
+  version = "1.59.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "nominal-io";
+    repo = "nominal-client";
+    tag = "v${version}";
+    hash = "sha256-wWy7i9lWZzDCx3cPyViLp7r7hSSDQbqdSUABX7L0LSE=";
+  };
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    requests
+    conjure-python-client
+    nominal-api
+    nominal-api-protos
+    python-dateutil
+    pandas
+    typing-extensions
+    click
+    pyyaml
+    tabulate
+    ffmpeg-python
+  ];
+
+  nativeCheckInputs = [
+    coverage
```

Comment:
This is a runtime dependency.

Reply:
Then you should be able to remove it completely.

Reply:
wth, I swear I got a pytest error earlier when I removed coverage? Maybe it was because I was using `pytestFlags` instead of `disableTestPaths` or something? idk, but great--was able to remove.